### PR TITLE
Document ordered list style

### PR DESCRIPTION
**Pull request by jdbaldry**
Signed-off-by: Jack Baldry <jack.baldry@grafana.com>

### CODE REVIEW

**Comment by josmperez** [comment]
Hi Jack - My personal preference is to *not* give doc authors the choice to use either "lazy" or "fully numbered" lists, because it's easier for me to edit when there is only one correct way. I think this is our current guidance and not sure why it's changing, Is this something we can poll the TWs on?

File: docs/sources/style-guide/style-conventions/index.md [hunk 1]
```diff
@@ -90,14 +90,45 @@ Use the following techniques to enhance the scannability of content.
 - Use short, bulleted lists.
 - Use headings to divide content.
 
-## Unordered lists
+## Lists
+
```
Comment (knylander-grafana) [inline docs/sources/style-guide/style-conventions/index.md:94]: Should we include term and definition lists here?

File: docs/sources/style-guide/style-conventions/index.md [hunk 2]
```diff
@@ -90,14 +90,45 @@ Use the following techniques to enhance the scannability of content.
...
 
 Refer to the following guidelines when you write unordered lists.
 
-For more guidance, refer to [Lists](https://developers.google.com/style/lists) in the [Google developer style guide](https://developers.google.com/style/).
-
 - Begin list items with a capital letter unless there is a strong reason not to. For example, when you list case-sensitive parameters.
-- If they are complete sentences, end list items with a period. If one item in a list ends with a period, then apply periods to all items in the list.
+- If they're complete sentences, end list items with a period. If one item in a list ends with a period, then apply periods to all items in the list.
+
+For more guidance, refer to [Lists](https://developers.google.com/style/lists) in the [Google developer style guide](https://developers.google.com/style/).
```
Comment (knylander-grafana) [inline docs/sources/style-guide/style-conventions/index.md:131]:
```suggestion
For more guidance, refer to [Lists](https://developers.google.com/style/lists) in the [Google developer style guide](https://developers.google.com/style/).

### Definition lits

Definition lists often used for providing a list of a term and indented definition. For example, you can use a definition list to document commands and their meanings: 

Feature
: Creates a new feature 

Features
: Creates more than one feature
: Sample additional features

To create a definition list, add the term on a new line then add a new line with a colon (`:`) along with the definition.

 For additional information, refer to the [Definition Lists](https://www.markdownguide.org/extended-syntax/#definition-lists) from the Markdown Guide.  
```

**Review by knylander-grafana — APPROVED** [review]
Thank you for adding content, @jdbaldry.  I've added some suggestions to your content to include definition lists.

File: docs/sources/style-guide/style-conventions/index.md [hunk 3]
```diff
@@ -90,14 +90,45 @@ Use the following techniques to enhance the scannability of content.
 - Use short, bulleted lists.
 - Use headings to divide content.
 
-## Unordered lists
+## Lists
+
+### Ordered lists
```
Comment (josmperez) [inline docs/sources/style-guide/style-conventions/index.md:95]: I don't want this comment to be a blocker, and I believe I said it somewhere else (Slack maybe?)... Adding guidance for nested lists is awesome, but personally I would rather not give people too different paths and let them decide if their list is big enough to use one path over another.

**Review by josmperez — APPROVED** [review]

File: docs/sources/style-guide/style-conventions/index.md (hunk 3 above)
Comment (jdbaldry) [reply docs/sources/style-guide/style-conventions/index.md:95]: I'd also support that behavior but the outcome of the style roundtable was to follow this style. That being said, it wasn't brought to a poll. I'll add it for a discussion in the next meeting.

**Pull request merged**